tests/test_tensorflow_sparse_data_api.py

```diff
@@ -213,118 +257,140 @@ def test_except_with_diff_number_of_batch_x_y_rows_empty_record(
             data=spoiled_data,
             batch_size=BATCH_SIZE,
             sparse=True,
+            num_of_attributes=num_of_attributes,
         )
         ingest_in_tiledb(
             uri=tiledb_uri_y,
             data=create_sparse_array_one_hot_2d(*dataset_shape_y),
             batch_size=BATCH_SIZE,
             sparse=False,
+            num_of_attributes=num_of_attributes,
         )
 
         with tiledb.open(tiledb_uri_x) as x, tiledb.open(tiledb_uri_y) as y:
             tiledb_dataset = TensorflowTileDBSparseDataset(
-                x_array=x, y_array=y, batch_size=BATCH_SIZE
+                x_array=x,
+                y_array=y,
+                x_attribute_names=[
+                    "features_" + str(attr) for attr in range(num_of_attributes)
+                ],
+                y_attribute_names=[
+                    "features_" + str(attr) for attr in range(num_of_attributes)
+                ],
+                batch_size=BATCH_SIZE,
             )
             with pytest.raises(Exception):
-                model.fit(tiledb_dataset, verbose=0, epochs=2)
+                next(tiledb_dataset)
 
-    def test_except_with_multiple_nz_value_record_of_batch_x_y_rows(
```

Comment:
Is this test of no use anymore?

Reply:
Was this test correct? I couldn't get was it testing based on it's name and zero number of assertions.

Reply:
The test was testing some limitations of sparse implementations during batching. If I recall correctly on what happens when we have more than one non zero value in a sparse record. The test had to succeed thus there are no assertions statements. In any case let's remove for now and we revisit the necessity of it in the future